app/Http/Controllers/MembersController.php

```diff
@@ -69,7 +70,27 @@ public function edit(Member $member)
 	# Update member in DB
 	public function update(Member $member, Requests\MemberRequest $request)
 	{
-		$member->update($request->except("roles"));
+		$member->update($request->except(["skills", "roles"]));
+
+		// Update skills
+		$skills = $request->input('skills'); // this is an array with ids of existing skills (as strings!) and string tags of new skills
+
+		$skill_ids = [];
+		foreach ($skills as $skill_id) {
```

Comment:
This fails if you leave skills empty.

repro:
- Edit a member
- Make sure their skils are empty
- see: Broken

I know not allot of people have no skills. But still I am a member within AAS2 :p

Reply:
Nice one. Turns out the select input is `null` instead of `[]` when empty. Easy fix: `$skills = $request->input('skills') ?: [];`. Yay for ternary operators.

Reply:
Could you use `??` which is the `null coalescing` operator and this is exactly what it is for. If for some reason the `$request->input('skills')` returns an empty string it won't default to `[]` which is not preferable I think.